src/index.tsx

```diff
@@ -169,7 +170,7 @@ export default class InlineSVG extends React.PureComponent<IProps, IState> {
         replaceableAttributes.forEach(r => {
           const attribute = attributes.find(a => a.name === r);
 
-          if (attribute) {
+          if (attribute && !(linkAttributes.includes(r) && attribute.value[0] !== '#')) {
```

Comment:
I'm not including a polyfill for `includes` in the build, so it's better to use `indexOf`.
Also, I think it would be better to use `indexOf` in the `attribute.value` check too.